webview/src/setup/components/remotes/ShowExtension.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styles from './styles.module.scss'
+import { Icon } from '../../../shared/components/Icon'
+import { Extensions } from '../../../shared/components/icons'
+
+export const ShowExtension: React.FC<{ id: string; name: string }> = ({
+  id,
+  name
+}) => {
+  const idQuery = `"@id:${id}"`
+
+  return (
+    <p>
+      <Icon
+        icon={Extensions}
+        width={16}
+        height={16}
+        className={styles.infoIcon}
+      />{' '}
+      View the{' '}
+      <a
+        href={`command:workbench.extensions.search?${encodeURIComponent(
+          idQuery
+        )}`}
```

Comment:
[F] This took a long time to workout. I actually had to visit the VS Code codebase to copy the expected format.

Finally found the format at 
https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/welcomeGettingStarted/common/media/extensions-web.svg#L44